### DISCUSSION TITLE
Official ProfSynapse GPTs?

### DISCUSSION THREAD

**Discussion by 3x3cut0r** [ProfSynapse/Professor-Synapse#18] [General]
Does anyone build a ProfSynapse [GPTs](https://openai.com/blog/introducing-gpts)?
Are there any best practices?
Is there an "official" public available ProfSynapse GPTs out there?
I build one: [Professtor Synapse GPTs](https://chat.openai.com/g/g-Nzpa4t3FG-professor-synapse)
Tell me your thoughts

Thank you

**Comment by ProfSynapse** [comment]
Don't worry we're working on it! Just got access yesterday. Follow our instagram @synapticlabs for when we drop it

**Comment by HansP95** [comment]
Share your GPTs on [https://gptspedia.io](url) and apply to be GPT of the week! ❤️

**Comment by ProfSynapse** [comment]
Here is it! https://chat.openai.com/g/g-ucpsGCQHZ-professor-synapse

**Comment by 3x3cut0r** [comment]
Why I can't find it in the GPTs store?
